app/error_handlers.py

```diff
@@ -47,6 +47,8 @@ def create_error_middleware(overrides):
             return await connection_error(request, ex.os_error.strerror)
         except ContentTypeError as ex:
             return await payload_error(request, str(ex.request_info.url))
+        except KeyError:
+            return await response_error(request)
```

Comment:
Catching any `KeyError` like this _may_ result in losing the cause of the error, and make it impossible to differentiate from `ClientResponseError` conditions without logging being present. Is this acceptable, or is logging needed?